.ci/Manifest.toml

```diff
@@ -28,19 +28,19 @@ uuid = "2a0f44e3-6c83-55bd-87e4-b1978d98bd5f"
 
```

Comment:
To be clear, I don't think this is a good long-term solution.  I'll need to update BinaryBuilderBase for #3229 for example (we need https://github.com/JuliaPackaging/BinaryBuilderBase.jl/pull/156).  What if we move [`csl_paths(host_platform)`](https://github.com/JuliaPackaging/BinaryBuilderBase.jl/blob/4b262c538cb22283f0b2e88e1777efac35fa135c/src/Runner.jl#L925-L926) at the beginning of `LD_LIBRARY_PATH` instead?  @staticfloat?